http/Client.http

```diff
@@ -0,0 +1,5 @@
+http://localhost:80/todos2/todos
+
+###
+
+http://localhost:80/todos/todos
```

Comment:
What is this?

Reply:
presumably it can be deleted...

Reply:
> What is this?

Client.http is a client from IntelliJ, I forgot to remove it :S